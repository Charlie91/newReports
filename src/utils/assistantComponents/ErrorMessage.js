import React from 'react';

const ErrorMessage = (props) => (
    <div className="message text-white bg-danger text-center card">
        <div className="card-body card-block">
            {props.text}
        </div>
    </div>
);
export default ErrorMessage;