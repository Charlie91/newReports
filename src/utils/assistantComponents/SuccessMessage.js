import React from 'react';

const SuccessMessage = (props) => (
    <div className="message text-white bg-success text-center card">
        <div className="card-body card-block">
            {props.text}
        </div>
    </div>
);

export default SuccessMessage;