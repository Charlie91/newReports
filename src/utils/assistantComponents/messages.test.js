import React from 'react';
import renderer from 'react-test-renderer';
import ErrorMessage from './ErrorMessage';
import SuccessMessage from './SuccessMessage';


describe('Messages are shown correctly', () => {
    it('renders correctly', () => {
        const tree = renderer
            .create(<ErrorMessage text="Текст ошибки"/>)
            .toJSON();
        expect(tree).toMatchSnapshot();
    });

    it('renders correctly', () => {
        const tree = renderer
            .create(<SuccessMessage text="Текст успешного сообщения"/>)
            .toJSON();
        expect(tree).toMatchSnapshot();
    });
});

