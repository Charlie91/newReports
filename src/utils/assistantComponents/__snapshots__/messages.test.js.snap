// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Messages are shown correctly renders correctly 1`] = `
<div
  className="message text-white bg-danger text-center card"
>
  <div
    className="card-body card-block"
  >
    Текст ошибки
  </div>
</div>
`;

exports[`Messages are shown correctly renders correctly 2`] = `
<div
  className="message text-white bg-success text-center card"
>
  <div
    className="card-body card-block"
  >
    Текст успешного сообщения
  </div>
</div>
`;
